lib/rake/firebase.rake

```diff
@@ -22,6 +22,11 @@ namespace :firebase do
             "To upload the firebase rules you built using `rake firebase:compile_rules`, you will need to\n"\
             "set `use_my_apps: true` in locals.yml and then run `rake package:apps:symlink`.\n\n"
         end
+        if !system('ls -L public/blockly')
```

Comment:
Why not [`File::exist?`](https://ruby-doc.org/core-2.2.0/File.html#method-c-exist-3F)